src/model/core/co2.jl

```diff
@@ -0,0 +1,122 @@
+@doc raw""" 
+    co2!(EP::Model, inputs::Dict)
+
+This function creates expressions to account for CO2 emissions as well as captured and sequestrated 
+CO2 from thermal generators. It also has the capability to model the negative CO2 emissions 
+from bioenergy with carbon capture and storage. 
+
+***** Expressions *****
+
+For thermal generators which combust fuels (e.g., coal, natural gas, and biomass),
```

Comment:
I don't think this adequately addresses @gschivley's comments. I think it would be clearer to just state what the module is doing, rather than giving background about whether or not biofuels are carbon neutral. Here is my suggestion:

For thermal generators which combust fuels (e.g., coal, natural gas, and biomass), the net CO2 emission to the environment is a function of fuel consumption, CO2 emission factor, CO2 capture fraction, and whether the feedstock is biomass. Biomass is a factor in this equation because biomass generators are assumed to generate zero net CO2 emissions, or negative net CO2 emissions in the case that the CO2 they emit is captured and sequestered underground.

Reply:
That sounds great! I'll replace that with your comment!